TodoListService/Controllers/TodoListController.cs

```diff
@@ -155,15 +159,15 @@ public IEnumerable<TodoItem> Get()
                 }
                 catch (AdalException ex)
                 {
-                    if (ex.ErrorCode == "temporarily_unavailable")
+                    if (ex.ErrorCode == SERVICE_UNAVAILABLE)
```

Comment:
Careful with retries. This seems fine since it's limited, but generally it's not a good idea. Msg me for adal error handling doc where we broke down all the cases.